tests/script-based-pre/verify_std_cmd/verify_std.sh

```diff
@@ -85,7 +100,7 @@ cat ${TMP_DIR}/std_lib.rs >> ${TMP_DIR}/library/std/src/lib.rs
 
 echo "[TEST] Run kani verify-std"
 export RUST_BACKTRACE=1
-kani verify-std -Z unstable-options "${TMP_DIR}/library" --target-dir "${TMP_DIR}/target"
+kani verify-std -Z unstable-options "${TMP_DIR}/library" --target-dir "${TMP_DIR}/target" -Z stubbing
```

Comment:
Why `-Z unstable-options` doesn't include stubbing? Should we also add `-Z function-contracts`

Reply:
We can add `-Z function-contracts` with an example once Jai's PR has been merged (#3153).